modules/client/examples.md

```diff
@@ -475,21 +476,27 @@ const client: Client = new Client(context);
 
 // Use an already existing ERC20 token
 const pluginInitParams1: ITokenVotingPluginInstall = {
-  settings: {
+  votingSettings: {
     minDuration: 60 * 60 * 24 * 2, // seconds
-    minTurnout: 0.25, // 25%
-    minSupport: 0.5, // 50%
+    minParticipation: 0.25, // 25%
+    supportThreshold: 0.5, // 50%
+    minProposerVotingPower: BigInt("5000"),
```

Comment:
Is this parameter optional? If by default it is 0, I would add a comment to the right (`Default: 0`)